worker/buildbot_worker/commands/transfer.py

```diff
@@ -107,7 +107,8 @@ def start(self):
         self._reactor.callLater(0, self._loop, d)
 
         def _close_ok(res):
-            self.fp = None
+            if self.fp:
+                self.fp.close()
```

Comment:
I think it is always safe to break the object links and help gc by forcefully put stuff to None when you dont use it anymore.